.github/workflows/ci.yml

```diff
@@ -420,15 +420,25 @@ jobs:
     - name: Install dependencies
       run: |
         brew update
-        brew install libmad libpng jpeg-turbo sdl2
+        brew install libmad libpng sdl2 jpeg-turbo
+    - name: Add support files
+      run: |
+        mkdir -p Contents/Frameworks Contents/MacOS Contents/Resources/data Contents/Resources/images Contents/Resources/sounds
+        ln -s /usr/local/opt/libmad/lib/libmad.0.dylib Contents/Frameworks/
+        ln -s /usr/local/opt/libpng/lib/libpng16.dylib Contents/Frameworks/
+        ln -s /usr/local/opt/sdl2/lib/libSDL2.dylib Contents/Frameworks/
+        ln -s /usr/local/opt/jpeg-turbo/lib/libturbojpeg.0.dylib Contents/Frameworks/
+        touch Contents/Resources/credits.txt
+        printf 'conversation "default intro"\n\tname' > Contents/Resources/data/start.txt
```

Comment:
I've been been doing native builds on Linux and web builds on macOS for a while now, but back when I did native macOS builds I did them with Xcode by changing Frameworks and Header Search Paths and Library Search Paths manually as mentioned in readme-developer.txt. I don't know if these instructions currently work.

Reply:
All these steps do is create a minimal app package that the binary expects. The libraries are symlinked to `Frameworks` from where Homebrew installs them, a `credits.txt` [is created](https://github.com/endless-sky/endless-sky/blob/master/source/Files.cpp#L135), and enough of a `data/start.txt` for `test_parse.sh` to pass (avoiding "Error: the "default intro" conversation must contain a "name" node.").

Reply:
> All these steps do is create ... enough of a `data/start.txt` for `test_parse.sh` to pass (avoiding "Error: the "default intro" conversation must contain a "name" node.").

The point of the parsing test is two-fold:
1) the binary can parse the data content
2) the game content is free of parsing errors.

With your changes, it seems this test no longer can perform the second function for macOS. Based on the changes you made to make this test pass, it seems that, if one were to modify non-compiled content such as a mission definition, they would need to perform a build in XCode before re-executing the binary, or else the changes would not be observed by the executable.

If that's _already_ the required workflow on macOS, that's fine, but if it isn't, I don't think we should change it.

Reply:
Based on feedback from Discord, it seems like this workflow change is fine.

We can restore the ability of the macOS parse job to parse content from the repo by modifying the test script to take an optional parameter specifying the resources directory, and set that parameter in the CI test. **_That change can be done separately_**

Reply:
I've made a change so that it adds symlinks to where the game data has been checked out to, which I didn't initially notice was available. The binary should be able to notice changes in game data without recompilation.